app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -388,6 +393,32 @@ class SuggestedEditsTasksFragment : Fragment() {
         }
     }
 
+    private fun showSurveyDialog(
```

Comment:
Since this dialog will not be reused in other places, can we make this even simpler, just like this?

Please make sure to add a check for the preference for the one-time popup purpose. 

https://github.com/wikimedia/apps-android-wikipedia/blob/89412c1ebc52e1f1b7fceb9db38dc8db8cde0409/app/src/main/java/org/wikipedia/main/MainFragment.kt#L547-L557

Reply:
done.